packages/discord.js/src/structures/Interaction.js

```diff
@@ -225,6 +225,15 @@ class Interaction extends Base {
       ComponentType[this.componentType] === ComponentType.SelectMenu
     );
   }
+
+  /**
+   * Indicates whether this interaction can be replied to or not.
+   * @returns {boolean}
+   */
+  isRespondable() {
+    const thisType = InteractionType[this.type];
+    return thisType !== InteractionType.Ping && thisType !== InteractionType.ApplicationCommandAutocomplete;
```

Comment:
shorter = better?
```suggestion
	return ![InteractionType.Ping, InteractionType.ApplicationCommandAutocomplete].includes(this.type)
```

Reply:
You'd have to either keep the `thisType` variable or use `InteractionType[this.type]` in the `includes()`

Reply:
Yeah I messed that up, edited the comment now